gobblin-temporal/src/main/java/org/apache/gobblin/temporal/dynamic/WorkforcePlan.java

```diff
@@ -78,7 +78,7 @@ protected UnrecognizedProfile(ScalingDirective directive) {
   }
 
   private final WorkforceProfiles profiles;
-  private final WorkforceStaffing staffing;
+  @Getter private final WorkforceStaffing staffing;
```

Comment:
let's keep this encapsulated (`private`) and only glimpse the values indirectly through`calcStaffingDeltas`